src/python/zquantum/qcbm/ansatz.py

```diff
@@ -3,7 +3,15 @@
 
 def get_qcbm_ansatz(n_qubits, n_layers, topology):
     """
-    TODO
+    Creates a circuit template for the QCBM ion trap ansatz.
```

Comment:
I feel like circuit template is not super clear to users - unless we define what we mean. At least, I was confused by this when I started using the platform, could be just me :)

Reply:
Oh yes, we'll replace it with the Ansatz interface :)